python/ray/reporter.py

```diff
@@ -9,6 +9,12 @@
 import subprocess
 from concurrent import futures
 
+import sys
+if "psutil" not in sys.modules:
+    psutil_path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)), "psutil_files")
+    sys.path.insert(0, psutil_path)
+
 try:
     import psutil
```

Comment:
Let's make sure to change all occurrence of `import psutil` and `import setproctitle` to `import ray.thirdparty.psutil import psutil`